modules/ossm-validate-smcp-kiali.adoc

```diff
@@ -0,0 +1,52 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-create-smcp.adoc
+////
+:_content-type: PROCEDURE
+[id="ossm-validate-control-plane-kiali_{context}"]
+= Validating your SMCP installation with Kiali
+
+You can use the Kiali console to validate your {ProductShortName} installation. The Kiali console offers several ways to validate your {ProductShortName} components are deployed and configured properly.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with cluster-admin rights. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+. Navigate to *Networking* -> *Routes*.
+
+. On the *Routes* page, select the control plane project, for example `istio-system`, from the *Namespace* menu.
++
+The *Location* column displays the linked address for each route.
++
+. If necessary, use the filter to find the route for the Kiali console. Click the route *Location* to launch the console.
+
+. Click *Log In With OpenShift*.
++
+When you first log in to the Kiali Console, you see the *Overview* page which displays all the namespaces in your service mesh that you have permission to view.  When there are multiple namespaces shown on the *Overview* page, Kiali shows namespaces with health or validation problems first.
++
+.Kiali Overview page
+image::ossm-kiali-overview.png[Kiali Overview page showing istio-system]
++
+If you are validating the console installation and namespaces have not yet been added to the mesh, there might not be any data to display other than `istio-system`. The tile for each namespace displays the number of labels, the *Istio Config* health, the number of and *Applications* health, and *Traffic* for the namespace.
+
+. To view the topology, click the *Graph* page, select the *Namespace* where you installed the `ServiceMeshControlPlane` from the menu, for example `istio-system`.
```

Comment:
This paragraph and the preceding one had me slightly confused. The previous paragraph says, "If you are validating...". So, OK, you might not have any apps and you just want to see that the Control Plane is installed.
But this second paragraph is separate - so it isn't clear this is meant as part of the validation of the control plane install. It says "to view the topology", but it looks like this is only to validate the install because it says to change the Namespace to the SMCP location.

To view topology, you would need to select some/all the namespaces in the SMMR (i.e. the namespaces where the apps are). Otherwise, just viewing the namespace where the SMCP is installed won't show you the topology of your mesh - it will only show you the nodes within the control plane itself.

Perhaps this is nitpicky, but its the only thing I could find to comment on - everything else is good :)